src/Permissions/Widgets/SidebarRow.vala

```diff
@@ -68,8 +72,14 @@ public class Permissions.SidebarRow : Gtk.ListBoxRow {
             }
         }
 
-        var description = string.joinv (", ", current_permissions.data);
-        description_label.label = "<small>%s</small>".printf (description);
-        set_tooltip_text (description);
+        if (current_permissions.length > 0) {
+            var description = string.joinv (", ", current_permissions.data);
```

Comment:
I'm sure that it is not i18n friendly as not all language use this specific comma character for this